data.sql

```diff
@@ -1,5 +1,85 @@
-/* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ('Agumon',  '2020-02-03', 0, TRUE, 10.23 );
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ('Gabumon',  '2018-11-15', 2, TRUE, 8 );
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ('Pikachu','2021-01-7', 1, FALSE, 15.08 );
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ('Devimon','2017-05-12', 5, TRUE, 11 );
+
+
+/*********************** Insert new Data **************************/
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ('Charmander','2020-02-08', 0, FALSE, -11 );
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ('Plantmon','2021-11-15', 2, TRUE, -5.7 );
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ('Squirtle','1994-04-02', 3, FALSE, -12.13 );
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ('Angemon','2005-06-12', 1, TRUE, -45 );
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ('Boarmon','2005-06-07', 7, TRUE, 20.4 );
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ('Blossom','1998-10-13', 3, TRUE, 17 );
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES ('Ditto','1998-05-14', 4, TRUE, 22 );
```

Comment:
- [ ] I believe that you made a typo here. The birth year for animal `Ditto` should be `2022`